dspace/config/modules/identifiers.cfg

```diff
@@ -0,0 +1,43 @@
+#----------------------------------------------------------------------#
+#---------------------IDENTIFIER CONFIGURATIONS------------------------#
+#----------------------------------------------------------------------#
+# These configs are used for additional identifier configuration such  #
+# as the Show Identifiers step which can "pre-mint" DOIs and Handles   #
+#----------------------------------------------------------------------#
+
+# Should DOIs be minted for (future) registration at workspace item creation?
```

Comment:
I noticed this `identifiers.submission.register=true` also seems to work for Handles.  So, I think we need to update these comments to make it clear this setting is NOT specific to DOIs.

Reply:
Yes good point, this is one part of the code that is wider than just DOIs.